test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -305,6 +305,18 @@ public void TestPfiBinaryClassificationOnSparseFeatures(bool saveModel)
 
             Done();
         }
+
+        [Fact]
+        public void TestBinaryClassificationWithoutCalibrator()
```

Comment:
The test does not Assert anything. Can you please include Asserts for the relevant results that this test is supposed to verify?